### ISSUE TITLE
DSA 2.1.11.1 longest increasing subsequence

### ISSUE THREAD

**Issue by vikhyatsingh123** [girlscript/winter-of-contributing#3605]
## Description

Explain and implement Longest Increasing Subsequence Size of an array.

## Task
- [ ] Audio
- [ ] Video
- [ ] Documentation

## Note
- While requesting the issue to be assigned, please mention your **NAME & BATCH NUMBER**.
- **Mention the type of contribution you want to make**.
- Kindly refer to the [Issue Guidelines](https://github.com/girlscript/winter-of-contributing/blob/DSA/DSA/CONTRIBUTING.md#issues).
- **Do not use the assign command for this issue. Follow the above procedure.**
- In case you see that the issue has already been assigned, try to request a different task within the same issue.
- Changes should be made inside the ````DSA/directory```` & ````DSA```` branch.
- Name of the file should be the same as the issue.
- This issue is only for 'GWOC' contributors of the 'DSA' Domain.

### Domain

DSA

### Type of Contribution

Audio, Video, Documentation

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.

**Comment by Harshil-Jani** [comment]
@vikhyatsingh123 Hey! Can I take this up for Documentation Part ? 
**Harshil Jani**
**DSA-Batch2**

**Comment by mehabhalodiya** [comment]
@goelesha Please assign this issue to @Harshil-Jani

**Comment by vikhyatsingh123** [comment]
@Harshil-Jani I am assigning to you as you have commented first. But from next time, don't open PR before getting assigned.

**Comment by Harshil-Jani** [comment]
> @Harshil-Jani I am assigning to you as you have commented first. But from next time, don't open PR before getting assigned.

I have not created any PR yet :sweat_smile: It just says, I mentioned this Issue  in another Issue.

**Comment by vikhyatsingh123** [comment]
@Harshil-Jani Remove issue number that you have written in previous comment

**Comment by nikki-priyaHIT** [comment]
may i work on this issue?

**Comment by vikhyatsingh123** [comment]
@nikki-priyaHIT documentation part is assigned. You may take audio or video part

**Comment by Harshil-Jani** [comment]
@vikhyatsingh123 @mehabhalodiya @goelesha PR has been made ! If any changes are required do let me know :smile:

**Comment by rohit-goyal** [comment]
Hello My Name is Rohit Goyal 
Pls aasign the documentation Part of this to me
 I am From DSA Batch-2
